config/prod/prow/jobs/custom/test-infra.yaml

```diff
@@ -81,6 +81,22 @@ presubmits:
           secretName: test-account
 
 periodics:
+- cron: "*/3 * * * *" # Every 3 minutes
+  name: ci-knative-heartbeat
+  agent: kubernetes
+  decorate: true
+  cluster: "default"
+  spec:
+    containers:
+    - image: gcr.io/knative-tests/test-infra/prow-tests:stable
+      imagePullPolicy: Always
+      command:
+      - "runner.sh"
+      args:
+      - "go"
```

Comment:
Will it be failing since I assume there are no Go code without checking out a repo? Can we replace it with a simple `echo` command?

Reply:
Good catch! Would prefer it goes through cloneref path for better sanity checks, updated with cloneref

Reply:
Then my concern is your are running this job very frequently (every 3 minutes), and I don't think this `go test` job can be finished within 3 minutes..

Reply:
ack, updated

Reply:
Sorry for the back-and-forth here,  the main point of my last comment was actually `3 minutes` - since every time we update the `prow-tests` image, just the image download time will be more than 3 minutes so I'm worried it will be causing issues. 
Can we
1. Change it to be running less frequently (like every 10 minutes) and/or use a more lightweight test image?
2. Set its own resource requests to be much smaller, so that it can be scheduled more easily?

Reply:
How about:
1. change to per 5 minutes
2. just `ls /go/src/knative.dev/test-infra/go.mod` <== will fail if the file is not there

Reply:
I see what you meant, if there an image of recommendation?

And thanks for reminding, the resources requests will make this job not able to schedule, will update

Reply:
It should not be taking 3 minutes to download the image; it's cached

Reply:
I thought you meant downloading the repo

Reply:
I meant for downloading a *new* updated image :-)